### DISCUSSION TITLE
AirSim Mavlink Input for Digital Twin

### DISCUSSION THREAD

**Discussion by perfectflowltd** [microsoft/AirSim#4155] [Support Q&A]
### Overview:
It would be useful to use airsim/unreal as a digital twin of a real drone in flight

### Smaller Details: 
Virtual Airsim drone Mirrors real drone movement and positiob via MavLink

### Nature of Request:  

- Addition

  - 

## Why would this feature be useful?
Visualisation for office team no at test site.

**Comment by jonyMarino** [comment]
Hi and welcome @skytechltd!
You can create a script that uses mavsdk to get the vehicle position and then sets the position of the AirSim Drone through simSetVehiclePosition() using our API. You will need to configure the physics engine to use [`ExternaPhysicsEngine` in your settings.json](https://microsoft.github.io/AirSim/settings/#physicsenginename).
